src/java.desktop/share/classes/sun/java2d/pipe/RenderBuffer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, 2013, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file should be reverted. Other than two unneeded whitespace changes, there are no changes to this file (and if there were, changing the starting copyright date would be wrong).